src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -157,16 +158,22 @@ private TestCase MakeTestCaseFromXmlNode(XmlNode testNode)
         // public for testing
         public DiaNavigationData GetNavigationData(string className, string methodName)
         {
-            if (this.DiaSession == null) return null;
+            if (this.DiaSession == null || TargetAssembly == null) return null;
 
-            var navData = DiaSession.GetNavigationData(className, methodName);
+            var definingType = TargetAssembly.GetType(className);
+            if (definingType == null) return null;
+
+            var method = definingType.GetMethods().Where(o => o.Name == methodName).OrderBy(o => o.GetParameters().Length).FirstOrDefault();
+            if (method == null) return null;
```

Comment:
Now that I have researched further, I see other adapters do something similar, arbitrarily selecting a single overload of the method. This is the most you can do if you use DiaSession.